sklearn/model_selection/_search.py

```diff
@@ -833,10 +844,15 @@ class GridSearchCV(BaseSearchCV):
             - A string, giving an expression as a function of n_jobs,
               as in '2*n_jobs'
 
-    iid : boolean, default=True
+    iid : boolean, default=None
         If True, the data is assumed to be identically distributed across
         the folds, and the loss minimized is the total loss per sample,
-        and not the mean loss across the folds.
+        and not the mean loss across the folds. Default is True,
+        but will change to False in version 0.21.
```

Comment:
I think (particularly given that we tend to get complaints about deprecations) that it's worth adding a few words on why, or basically saying "We now consider the iid=True formulation to be optimising an incorrect cross validation objective."